galoy.yaml

```diff
@@ -29,13 +29,13 @@ coldStorage:
   minOnChainHotWalletBalance: 1000000
   minRebalanceSize: 10000000
   maxHotWalletBalance: 200000000
-  walletPattern: specter
-  onChainWallet: specter/coldstorage
-  targetConfirmations: 6
 bria:
   hotWalletName: dev
   queueNames:
     fast: dev
+  coldStorage:
+    walletName: cold
+    hotToColdRebalanceQueueName: dev
```

Comment:
same name at line 33? is that intentional? do they represent the same queue? if not we should have different name to make it explicit that it's not the same thing as to what is line 33 and line 35

Reply:
33 is a wallet name and 35 is a queue name. 35 + 38 are indeed the same.

Reply:
I think having different for queue and wallet can help convey the message on what is same and what is different. Using dev for dev doesn't help

Reply:
Yeah good point. These are auto-provisioned in bria as part of 'dev' mode. Can change the names there.